src/test/scala/org/ergoplatform/wallet/serialization/SerializationSpec.scala

```diff
@@ -30,4 +30,8 @@ class SerializationSpec
     checkSerializationRoundtrip(trackedBoxGen, TrackedBoxSerializer)
   }
 
+  property("ExtendedSecretKey serialization") {
+    checkSerializationRoundtrip(extendedSecretGen, ExtendedSecretKeySerializer)
```

Comment:
I noticed that in checkSerializationRoundtrip there is no `b shouldEqual recovered`, please add.

Reply:
Some instances cannot be correclty compared automatically. I think, checks in `checkSerializationRoundtrip` are sufficient and ensure serialized objects are equal.

Reply:
sound like a bug

Reply:
Fixed.